code/modules/modular_armor/modular.dm

```diff
@@ -426,17 +427,31 @@
 		to_chat(user, "Right click the helmet to toggle the visor internal lighting.")
 		to_chat(user, "Right click the helmet with paint to color the visor internal lighting.")
 
+/obj/item/clothing/head/modular/attack_hand(mob/living/user)
+	if(!storage)
+		return ..()
+	if(storage.handle_attack_hand(user))
+		return ..()
+
+/obj/item/clothing/head/modular/MouseDrop(over_object, src_location, over_location)
+	if(!storage)
+		return ..()
+	if(storage.handle_mousedrop(usr, over_object))
+		return ..()
```

Comment:
this is ugly as hell. Looks like there is no other solutions, but ew. Please do if(!storage || storage.handle_mousedrop(usr, over_object)) at least

Reply:
i just copied from jaeger torso lel

Reply:
also, suggest pls